repository/user_repository.go

```diff
@@ -4,20 +4,28 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
+	"time"
 
 	"github.com/jmoiron/sqlx"
+	"github.com/vkhichar/asset-management/contract"
+	"github.com/vkhichar/asset-management/customerrors"
 	"github.com/vkhichar/asset-management/domain"
 )
 
 const (
 	getUserByEmailQuery = "SELECT id, name, email, password, is_admin FROM users WHERE email= $1"
 	selectAllUsers      = "SELECT id, name, email, password, is_admin, created_at, updated_at FROM users"
+	getUserByIDQuery    = "SELECT id, name, email, password, is_admin, created_at, updated_at FROM users WHERE id = $1"
+	updateUserColumns   = "UPDATE users SET name = $1, password = $2, updated_at = $3 WHERE id = $4"
+	deleteUserById      = "DELETE FROM users WHERE id=$1"
```

Comment:
delete user should be soft delete here

Reply:
For user table we decided to permanently delete that user from table.